external-crates/move/crates/move-compiler/src/expansion/translate.rs

```diff
@@ -985,6 +989,7 @@ fn module_(
     context.pop_alias_scope(Some(&mut use_funs));
 
     let def = E::ModuleDefinition {
+        named_address_map: context.defn_context.named_address_mapping.clone().unwrap(),
```

Comment:
Please bind this outside like the rest of the arguments, just to make this more-consistent.